src/scripts/Connection.ts

```diff
@@ -0,0 +1,6 @@
+export function connection(mail: string, password: string): boolean {
+        if (mail != "christopherbatrier@gmail.com" || password != "password") {
```

Comment:
seriously ? you write you password ? :)

Reply:
No one except me might access to the application !! :)

Reply:
ok